README.md

```diff
@@ -71,6 +73,21 @@ vars:
     product_pass_through_columns: []
     product_variant_pass_through_columns: []
 ```
+
+### Disable Models
+This package was designed with the intention that users have all relevant Shopify tables being synced by Fivetran. However, if you are Shopify user that does not operate on returns or adjustments then you will not have the related source tables. As such, you may use the below variable configurations to disable the respective downstream models. All variables are `true` by default, only add the below configuration if you are wishing to disable the models:
+
+```yml
+# dbt_project.yml
+
+...
+vars:
+  shopify_source:
+    shopify__using_order_adjustment:  false  # true by default
+    shopify__using_order_line_refund: false  # true by default
+    shopify__using_order_refund:      false  # true by default
```

Comment:
so like 
```suggestion
  shopify__using_order_adjustment:  false  # true by default
  shopify__using_order_line_refund: false  # true by default
  shopify__using_order_refund:      false  # true by default
```

Reply:
This is a great point and something we should document for future updates. If the variables impact both source and transform then we should set the variables for a global config.